draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -500,21 +500,23 @@ Applications using Type Metadata defined in this specification are called "Consu
 
 All examples in this section are non-normative.
 
-Type Metadata about the value of a `vct` claim value can be retrieved as described in (#retrieving-type-metadata).
+The following is an example of an SD-JWT VC payload, containing a `vct` claim
+with the value `https://betelgeuse.example.com/education_credential`:
 
 ```json
 {
   "vct": "https://betelgeuse.example.com/education_credential",
-  "vct#integrity": "sha256-WRL5ca_xGgX3c1VLmXfh-9cLlJNXN-TsMk-PmKjZ5t0",
+  "vct#integrity": "sha256-oTxxE//FNomPebw6yH9aNuEPUjbiYqaxBj1ZgMMBsxHmjTOBDhRiVCUD2bhVlzYg",
```

Comment:
wait... `oTxxE//FNomPebw6yH9aNuEPUjbiYqaxBj1ZgMMBsxHmjTOBDhRiVCUD2bhVlzYg` decodes to 48 bytes, which doesn't track too well with `sha256` while `WRL5ca_xGgX3c1VLmXfh-9cLlJNXN-TsMk-PmKjZ5t0` is the 32 bytes one would expect. Why'd you change this value?

Reply:
```suggestion
  "vct#integrity": "sha256-WRL5ca_xGgX3c1VLmXfh-9cLlJNXN-TsMk-PmKjZ5t0",
```